pkg/lib/telemetry/telemetry.go

```diff
@@ -17,121 +17,175 @@ limitations under the License.
 package telemetry
 
 import (
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
-	"net/http"
+	"os"
 	"time"
 
 	"github.com/cortexlabs/cortex/pkg/consts"
 	"github.com/cortexlabs/cortex/pkg/lib/errors"
+	"github.com/cortexlabs/cortex/pkg/lib/parallel"
+	"github.com/getsentry/sentry-go"
+	"gopkg.in/segmentio/analytics-go.v3"
 )
 
-const (
-	eventPath = "/events"
-	errorPath = "/errors"
-)
+var _segment analytics.Client
+var _config *Config
 
-type Client struct {
-	url             string
-	http            http.Client
-	EnableTelemetry bool
-	OperatorID      string
+type Config struct {
+	Enable          bool
+	UserID          string
```

Comment:
nit: UserID -> ID?

Reply:
I used `UserID` since that's what it's called in segment/sentry/amplitude